app-modules/service-management/src/Filament/Resources/ServiceRequestTypeResource/Pages/EditServiceRequestTypeNotifications.php

```diff
@@ -53,7 +52,7 @@ class EditServiceRequestTypeNotifications extends EditRecord
 
     public static function canAccess(array $parameters = []): bool
     {
-        return ServiceRequestTypeNotifications::active() && parent::canAccess($parameters);
+        return parent::canAccess($parameters);
     }
```

Comment:
This whole method can be removed now since we don't need to overwrite it anymore.

Reply:
Done